Update README.md with examples from the QTEXAMPLES.zip resource

BrKn001 opened the pull request:
I have added the explanations for most of the QTEXAMPLES.zip projects from Piazza.

BrKn001 committed:
Update README.md with examples from the QTEXAMPLES.zip resource

BrKn001 committed:
Add files via upload

gokceuludogan commented on README.md:
```diff
@@ -207,6 +207,235 @@ Without the flexibility offered by subclassing, developers would be stuck using
 
 Thanks to subclassing, one can easily modify the offered elements to their needs with minimal work.
 
+# **Qt Examples**
```
I think we could move this section to a separate README file under examples.

BrKn001 replied:
I have moved the Qt Examples discussion to its own folder, and took the liberty to rename the folder to 'Examples' from 'QTEXAMPLES'.

gokceuludogan reviewed (approved):
Thank you for extending the repo with the examples. It would be more convenient to have a separate README file for the examples. We could merge this PR once the corresponding change has been done.

BrKn001 committed:
Reverted README.md to pre-examples version

BrKn001 committed:
Create README.md for the Examples folder

BrKn001 committed:
Create new file

BrKn001 committed:
Delete Examples directory

BrKn001 committed:
Create and add files to the Examples directory

'Examples' as a directory name fits better than 'QTEXAMPLES' given that this directory's connection to the QTEXAMPLES.zip resource on Piazza is explained within the README.md already.

BrKn001 committed:
Delete QTEXAMPLES directory
